### PR TITLE
WIP: Improved and components structure

### PR DESCRIPTION
**Pull request by dar-g**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -23,7 +29,58 @@ body {
...
+.error {
+  margin-bottom: 30px;
+}
+
+.inner-link:hover {
+  color: rgb(58, 96, 115);
+  font-weight: bold;
+}
+
+.main-page {
```
Comment (RuslanaJS) [inline src/App.css]: As i can see, this selector is a part of `MainPage` component, so i probable it's a good idea to have separate css/scss file for component itself and keep styles of it's content there. If you are familiar with the conception of css modules, let's use it here. BTW, react-create-app has this feature included by fefault, so you can check it here https://create-react-app.dev/docs/adding-a-css-modules-stylesheet

File: src/App.css [hunk 2]
```diff
@@ -23,7 +29,58 @@ body {
...
+}
+
+.btn {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  text-decoration: none;
+  padding: 10px;
+}
+input[type='button'], .btn {
```
Comment (RuslanaJS) [inline src/App.css:62]: If you choose to make empty rows between each selector, why not here?🙂

File: src/App.js [hunk 3]
```diff
@@ -1,12 +1,13 @@
 import './App.css';
 import MainPage from "./containers/MainPage/MainPage";
-import StartPageRouting from "./containers/StartPageRouting/StartPageRouting";
+import RegisterRouting from "./containers/RegisterRouting/RegisterRouting";
```
Comment (RuslanaJS) [inline src/App.js]: Please, keep using one type of quotes for import statements - singular or double, not both

File: src/containers/RegisterRouting/components/NameForm.jsx [hunk 4]
```diff
@@ -1,39 +1,36 @@
...
         props.setName(saveName);
     };
 
-    const inputNameRef = useRef();
-
-    const saveName = () => {
-        const nameFromInput = inputNameRef.current.value;
+   const saveName = () => {
+       const nameFromInput = inputNameRef.current.value;
+       console.log(typeof nameFromInput, nameFromInput);
```
Comment (RuslanaJS) [inline src/containers/RegisterRouting/components/NameForm.jsx]: You can leave console logs, when you logging some kind of errors of useful info for validation. If you need this for debugging only, don't forget to remove it before merge

File: src/containers/RegisterRouting/components/WeightForm.jsx [hunk 5]
```diff
@@ -22,10 +22,9 @@ function WeightForm(props) {
                     <input type="number" ref={inputWeightRef} />
                     <span>kg</span>
                 </label>
-                <input type="button" onClick={saveWeight} value="Save" />
             </div>
 
-            <Link to="/register/sex" onClick={onClickHandler}>Continue</Link>
+            <Link to="/register/gender" onClick={onClickHandler} className="btn">Continue</Link>
```
Comment (RuslanaJS) [inline src/containers/RegisterRouting/components/WeightForm.jsx]:
```suggestion
            <Link
               to="/register/gender"
               onClick={onClickHandler}
               className="btn"
              >
                  Continue
             </Link>
```

For long list of properties i'd like to format them this way for better readability

File: src/containers/RegisterRouting/components/GenderForm.jsx [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+import UserService from "../../../services/UserService";
+
+function GenderForm(props) {
+    // const inputGenderRef = useRef();
+
+    const onSelectChange = (event) => {
+        props.setGender(event.target.value);
+    }
+
+    // const saveGender = (event) => {
```
Comment (RuslanaJS) [inline src/containers/RegisterRouting/components/GenderForm.jsx]: If you don't need this comment anymore, please, remove it

**Review by RuslanaJS — CHANGES_REQUESTED** [review]
Good in overall. I have left minor comment to fix and one suggestion regarding html formatting.

File: src/App.css (hunk 1 above)
Comment (dar-g) [reply src/App.css]: Applied CSS modules

File: src/App.css (hunk 2 above)
Comment (dar-g) [reply src/App.css:62]: fixed

File: src/App.js (hunk 3 above)
Comment (dar-g) [reply src/App.js]: edited

File: src/containers/RegisterRouting/components/WeightForm.jsx (hunk 5 above)
Comment (dar-g) [reply src/containers/RegisterRouting/components/WeightForm.jsx]: applied

**Review by RuslanaJS — APPROVED** [review]

**Pull request merged**